spec/takeawaySpec.js

```diff
@@ -0,0 +1,28 @@
+'use strict'
+
+describe('Takeaway', function(){
+
+  var takeaway;
+
+  beforeEach(function() {
+    takeaway = new Takeaway();
+  });
+
+  it('lists the dishes with prices', function() {
+    expect(takeaway.takeawayMenu()).toEqual({dish: 'Burrito', price: 7})
+  });
+
+  it('can select a dish', function() {
+    takeaway.order('Burrito')
+    expect(takeaway.customerItems).toContain({ dish: 'Burrito', price: 7})
+  });
+
+  it('can select multiple dishes', function() {
+    takeaway.order('Taco')
+    takeaway.order('Cerveza')
+    expect(takeaway.customerItems).toContain({ dish: 'Burrito', price: 7}, {dish: 'Taco', price: 5}, { dish: 'Cerveza', price: 6})
+  });
```

Comment:
Would you expect the customerItems to include a Burrito dish when the set up only orders Taco and Cerveza?